.gitlab/package_build/linux.yml

```diff
@@ -80,6 +80,14 @@
   before_script:
     - export RELEASE_VERSION=$RELEASE_VERSION_7
 
+.agent_fips_build:
+  variables:
+    AGENT_MAJOR_VERSION: 7
+    FLAVOR: fips
+  before_script:
+    - export RELEASE_VERSION=$RELEASE_VERSION_7
+    - DD_GO_TOOLCHAIN="msgo" source /root/.bashrc
```

Comment:
What does this do? We could set this variable inside Omnibus where it's specifically needed, given that there's already logic in there to handle FIPS mode; and we don't source `.bashrc` in the other builds as far as I can tell.

Reply:
this changes [the PATH](https://github.com/DataDog/datadog-agent-buildimages/blob/main/setup_go.sh#L37-L43) for use the [microsoft/go](https://github.com/microsoft/go/tree/microsoft/main) toolchain for building a fips compliant agent, if we aren't [sourcing bashrc in the runner](https://github.com/DataDog/datadog-agent/pull/31004#discussion_r1850039221) by default then we'll need the before_script to do this

Reply:
fixed in https://github.com/DataDog/datadog-agent/pull/31004/commits/dada6cbb431637c499d1407f1c43bb5a7b70baaf